api/v1/api.go

```diff
@@ -456,8 +463,14 @@ func (api *API) insertAlerts(w http.ResponseWriter, r *http.Request, alerts ...*
 			typ: errorInternal,
 			err: err,
 		}, nil)
+		if api.alertLCObserver != nil {
+			api.alertLCObserver.Rejected("Failed to create", validAlerts...)
```

Comment:
why this is rejecting?

Reply:
This is when `alerts.Put` failed. Since we don't end up recording the alert I considered it as rejected.